examples/js/loaders/GLTFLoader.js

```diff
@@ -23,6 +23,52 @@ THREE.GLTFLoader = ( function () {
 
 		load: function ( url, onLoad, onProgress, onError ) {
 
+			this._load( url, onLoad, onProgress, onError, false );
+
+		},
+
+		setCrossOrigin: function ( value ) {
+
+			this.crossOrigin = value;
+			return this;
+
+		},
+
+		setPath: function ( value ) {
+
+			this.path = value;
+			return this;
+
+		},
+
+		setResourcePath: function ( value ) {
+
+			this.resourcePath = value;
+			return this;
+
+		},
+
+		setDRACOLoader: function ( dracoLoader ) {
+
+			this.dracoLoader = dracoLoader;
+			return this;
+
+		},
+
+		parse: function ( data, path, onLoad, onError ) {
+
+			this._parse( data, path, onLoad, onError, false );
+
+		},
+
+		getParser: function ( url, onLoad, onProgress, onError ) {
+
+			this._load( url, onLoad, onProgress, onError, true );
+
+		},
+
+		_load: function ( url, onLoad, onProgress, onError, getParser ) {
```

Comment:
'_' suffix method name indicates private here. If we want real private method we can rewrite

```javascript
function loadGLTF( gltfLoader, url, onLoad, onProgress, onError, getParser ) {

    ...

}
```

Reply:
Renamed `getParser` to `parserOnly`, thanks.